src/store/configureStore.js

```diff
@@ -13,13 +13,13 @@ export default () => {
   return store;
 };
 
-//TODO: Also, but not in this PR (separate one) I recommend you to restucture the store 
+//TODO: Also, but not in this PR (separate one) I recommend you to restucture the store
```

Comment:
those comments are needed?

Reply:
yes, please ignore them, they're here only for keeping TODO's in code, they will disappear through time (and PR's)